harness/canary
# feat: add manage navigation component

This pull request brings manage navigation placeholder component.


<details>
<summary>Design</summary>

![image](https://github.com/user-attachments/assets/ef56f606-c984-4cd1-bc26-816b4e560d20)

</details>

Steps to check:
- open [preview](https://canary-git-feat-manage-navigation-pixelpoint.vercel.app/repos);
- make sure that everything looks and feels as expected;
